### PR TITLE
[VEN-2828]: ZKsync limits and refactor table structure

### PR DESCRIPTION
**Pull request by GitGuru7**

### CODE REVIEW

File: technical-reference/reference-technical-articles/xvs-bridge.md [hunk 1]
```diff
@@ -203,16 +203,48 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
...
-
-   | Transfer Path           | Single Send Limit | Single Receive Limit | Daily Send Limit | Daily Receive Limit |
-   |-------------------------|-------------------|----------------------|------------------|---------------------|
-   | BNB <--> Ethereum       | $100,000          | $102,000             | $1,000,000       | $1,020,000          |
-   | BNB <--> opBNB          | $10,000           | $10,200              | $50,000          | $51,000             |
-   | BNB <--> Arbitrum       | $20,000           | $20,400              | $100,000         | $102,000            |
-   | Ethereum <--> opBNB     | $10,000           | $10,200              | $50,000          | $51,000             |
-   | Arbitrum <--> Ethereum  | $20,000           | $20,400              | $100,000         | $102,000            |
-   | Arbitrum <--> opBNB     | $20,000           | $20,400              | $100,000         | $102,000            |
+- Single Send Limit
```
Comment (chechu) [inline technical-reference/reference-technical-articles/xvs-bridge.md]:
```suggestion
- Single Send Limit (source network in the first column, destination network in the first row)
```

File: technical-reference/reference-technical-articles/xvs-bridge.md [hunk 2]
```diff
@@ -203,16 +203,48 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
...
+   |          |   BNB     |   opBNB   |   Arbitrum   |   Ethereum   |   ZKsync  |
+   |----------|-----------|-----------|--------------|--------------|-----------|
+   | BNB      |     -     | $10,200   | $20,400      | $102,000     | $20,400   |
+   | opBNB    | $10,200   |    -      | $20,400      | $10,200      | $20,400   |
+   | Arbitrum | $20,400   | $20,400   |     -        | $20,400      | $20,400   |
+   | Ethereum | $102,000  | $10,200   | $20,400      |      -       | $20,400   |  
+   | ZKsync   | $20,400   | $20,400   | $20,400      | $20,400      |     -     |
+
+   
+- Daily Send Limit
```
Comment (chechu) [inline technical-reference/reference-technical-articles/xvs-bridge.md]:
```suggestion
- Daily Send Limit (source network in the first column, destination network in the first row)
```

**Review by chechu — COMMENTED** [review]
What do you think if we remove the tables for "Single Receive Limit" and "Daily Receive Limit" and we simply say that the Receive Limits are the Send Limits + 2% to have some margin in case of price fluctuations when the bridging transaction is being processed?

File: technical-reference/reference-technical-articles/xvs-bridge.md (hunk 2 above)
Comment (GitGuru7) [reply technical-reference/reference-technical-articles/xvs-bridge.md]: [19d3dcf](https://github.com/VenusProtocol/venus-protocol-documentation/pull/228/commits/19d3dcfcccb260bdaa1180ab95462bce08b40dd5)

File: technical-reference/reference-technical-articles/xvs-bridge.md (hunk 1 above)
Comment (GitGuru7) [reply technical-reference/reference-technical-articles/xvs-bridge.md]: [19d3dcf](https://github.com/VenusProtocol/venus-protocol-documentation/pull/228/commits/19d3dcfcccb260bdaa1180ab95462bce08b40dd5)

**Comment by GitGuru7** [comment]
> What do you think if we remove the tables for "Single Receive Limit" and "Daily Receive Limit" and we simply say that the Receive Limits are the Send Limits + 2% to have some margin in case of price fluctuations when the bridging transaction is being processed?

[19d3dcf](https://github.com/VenusProtocol/venus-protocol-documentation/pull/228/commits/19d3dcfcccb260bdaa1180ab95462bce08b40dd5)

**Review by chechu — APPROVED** [review]

**Pull request merged**